Populist Legal Power/common/laws/yMoG_PLP_citizenship.txt

```diff
@@ -214,13 +230,19 @@ law_racial_segregation = {
 		# state_radicals_from_sol_change_accepted_culture_mult = -0.1 # gone with 1.8
 		# state_loyalists_from_sol_change_accepted_culture_mult = 0.1 # gone with 1.8
 		# ALL NEW FROM PLP MOD:
-		country_radicalism_increases_violent_hostility_mult = 0.15
-		country_radicalism_increases_cultural_erasure_mult = 0.1
-		country_radicalism_increases_second_rate_citizen_mult = -0.05
-		country_radicalism_increases_full_acceptance_mult = -0.1
-		country_loyalism_increases_second_rate_citizen_mult = 0.1
-		country_loyalism_increases_full_acceptance_mult = 0.15
-		
+
+		country_radicalism_increases_violent_hostility_mult   =  0.4
+		country_radicalism_increases_cultural_erasure_mult    =  0.3
+		country_radicalism_increases_open_prejudice_mult      =  0.2
+		country_radicalism_increases_second_rate_citizen_mult =  0.1
+		country_radicalism_increases_full_acceptance_mult     = -0.1
+		
+		country_loyalism_increases_violent_hostility_mult   = -0.2
+		country_loyalism_increases_cultural_erasure_mult    = -0.15
+		country_loyalism_increases_open_prejudice_mult      = -0.1
+		country_loyalism_increases_second_rate_citizen_mult = -0.05
+		country_loyalism_increases_full_acceptance_mult     =  0.05
```

Comment:
Just something to consider - Rac. Seg. and Cul. Exc. don't have linear steps in their acceptance levels.

Maybe have a think about whether that might justify staggering these non-linearly in some way too.
- On RS, Open Prejudice is exclusively comprised of wrong religion + shared heritage or state religion + wrong heritage
- On CE, Open Prejudice is exclusively wrong religion + totally foreign

In both cases the lowest tiers basically don't exist, but you may want to have comparatively increased radicalism for those who are "so close, yet so far" from being accepted.